src/PhpPact/Standalone/Broker/Broker.php

```diff
@@ -245,13 +245,18 @@ public function publish(): void
                 $this->getArguments()
             )
         );
-        $runner->runBlocking();
 
-        if ($runner->getExitCode() !== 0) {
-            $this->logger->error($runner->getStderr());
-        }
+        try {
+            $runner->runBlocking();
 
-        $this->logger->debug($runner->getOutput());
+            $this->logger->info('out > ' . $runner->getOutput());
+            $this->logger->error('err > ' . $runner->getStderr());
+        } catch (\Exception $e) {
+            $this->logger->info('out > ' . $runner->getOutput());
+            $this->logger->error('err > ' . $runner->getStderr());
+
+            throw $e;
+        }
```

Comment:
```suggestion
try {
            $runner->runBlocking();
} finally {
            $this->logger->info('out > ' . $runner->getOutput());
            $this->logger->error('err > ' . $runner->getStderr());
}
```
I think this code is shorter and work the same.

Reply:
But we need re-throw exception also, not just output

Reply:
I tested with this code and I think the exception will be re-throwed.

```php
<?php

function throwException() {
    throw new Exception('test');
}

try {
    print("Outer Try\n");
    try {
        print("Try\n");
        throwException();
    } finally {
        print("Finally\n");
    }
} catch (\Throwable $th) {
    printf("Outer Catch: %s\n", $th->getMessage());
} finally {
    print("Outer Finally\n");
}

```

Output:

```php
Outer Try
Try
Finally
Outer Catch: test
Outer Finally
```